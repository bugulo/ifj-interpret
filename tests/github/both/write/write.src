.IPPcode21

DEFVAR GF@a
MOVE GF@a string@abc
WRITE GF@a
WRITE string@efg

CREATEFRAME
DEFVAR TF@a
MOVE TF@a int@123
WRITE TF@a
WRITE int@456

PUSHFRAME
MOVE LF@a bool@true
WRITE LF@a
WRITE bool@false

WRITE nil@nil

WRITE string@\010Hello\035World\010
