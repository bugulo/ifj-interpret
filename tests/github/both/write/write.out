abcefg123456truefalse
Hello#World
